operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -28,6 +28,16 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
   * Community Help Wanted doc issues
   * Documentation Handbook and Style Guide maintenance
 
+#### @sebastian.faase:
+
```

Comment:
Propose a slight tweak to this list @faase, let me know what you think?

```
- Training and Education guide
  - Enterprise rollout guide for self-serve post-sales technical onboarding
  - Improved Enterprise Edition feature usage and renewal rate
  - Reduced customer need for Mattermost support
- Liaison between product and sales/customer success to provide educational content to customers
  - Identify gaps and painpoints among customers
  - Prioritize customer education content generation
  - Attend customer calls undergoing enterprise onboarding
- Tone and writing style guide for customer education
```

^the training and education guide isn't yet live in docs, but it's in the works: https://github.com/mattermost/docs/pull/3526